pkg/standalone/standalone.go

```diff
@@ -48,8 +49,64 @@ const (
 	DaprRedisContainerName = "dapr_redis"
 )
 
+func isInstallationRequired(installLocation, requestedVersion string) bool {
+	destDir := daprDefaultLinuxAndMacInstallPath
+	if installLocation == "" {
+		if runtime.GOOS == daprWindowsOS {
```

Comment:
nit: I would not use nested if statement if possible :) 

```
if installLocation == "" && runtime.GOOS == daprWindowsOS {
..
```

Reply:
updated